### DISCUSSION TITLE
Resume training from checkpoint with new data

### DISCUSSION THREAD

**Discussion by mitchpudil** [Lightning-AI/pytorch-lightning#12845] [General]
I'm wanting to continue the training process for a model using new data.

I understand that you can continue training a Pytorch Lightning model e.g.

`pl.Trainer(max_epochs=10, resume_from_checkpoint='./checkpoints/blahblah.ckpt')` 

for example, if you last checkpoint is saved at epoch 5. But is there a way to continue training by adding different data?

**Comment by rohitgr7** [comment]
you can set up new dataloaders with the new data during the resume.
